test/CacheFactoryTest.php

```diff
@@ -64,13 +64,14 @@ public function testCacheChainContainsInitializedProviders() : void
             ],
         ];
 
-        $container = $this->prophesize(ContainerInterface::class);
-        $container->has('config')->willReturn(true);
-        $container->get('config')->willReturn($config);
-        $container->has(ArrayCache::class)->willReturn(false);
+        $container = $this->getMockBuilder(ContainerInterface::class)->onlyMethods(['has', 'get'])->getMock();
```

Comment:
Similar to the above: `$this->createMock()` suffices

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2